website/blog/2023-09-18-release-1.4.md

```diff
@@ -5,7 +5,7 @@ slug: podman-desktop-release-1.4
 authors: [jeffmaury]
 tags: [podman-desktop, release, kubernetes, openshift]
 hide_table_of_contents: false
-image: /img/blog/podman-desktop-release-1.4.webp
+image: /blog/img/podman-desktop-release-1.4/juggling.png
```

Comment:
you need to copy the image to the static folder here

Reply:
Oh I see: because the image that is in /blog/img/podman-desktop-release-1.4/juggling.png gets transformed into https://podman-desktop.io/assets/images/juggling-bfd34fcfc3b63b29752c5f16a53e36ad.png and the id is generated at build time.

Reply:
done